canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp

```diff
@@ -317,30 +322,36 @@ INT32 main(INT32 argc, CHAR* argv[])
         CHK(frame.frameData != NULL, STATUS_NOT_ENOUGH_MEMORY);
         frame.version = FRAME_CURRENT_VERSION;
         frame.trackId = DEFAULT_VIDEO_TRACK_ID;
-        frame.duration = 0;
+        frame.duration = (1 / DEFAULT_FPS_VALUE) * HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
```

Comment:
This will always be zero since 1 is integer.  Do this instead:
`frame.duration = (1.0 / DEFAULT_FPS_VALUE) * HUNDREDS_OF_NANOS_IN_A_MILLISECOND;`

Reply:
Or even this since lhs is UINT64:
`frame.duration = HUNDREDS_OF_NANOS_IN_A_MILLISECOND / DEFAULT_FPS_VALUE`

Reply:
Oops. Fixed.